qemu/tests/cfg/rdtsc_sync_test.cfg

```diff
@@ -0,0 +1,7 @@
+- rdtsc_sync_test:
+    type = rdtsc_sync_test
+    start_vm = no
+    auto_cpu_policy = "virttest"
+    source_file = 'rdtsc_sync_test/bz1975840.flat'
```

Comment:
Using bug ID is not a good idea, the file should have a clear name to explain what it does.

Reply:
> Using bug ID is not a good idea, the file should have a clear name to explain what it does.

Thanks and updated.